From: Marc Kleine-Budde <mkl@pengutronix.de>
Subject: [patch] make freetype-config $SYSROOT aware

Currently, freetype-config is not $SYSROOT aware, which is fixed by this
patch.

Signed-off-by: Marc Kleine-Budde <mkl@pengutronix.de>
---
# 20091221 rsc: submitted as https://savannah.nongnu.org/patch/index.php?7040

 builds/unix/freetype-config.in |   22 +++++++++++-----------
 1 file changed, 11 insertions(+), 11 deletions(-)

Index: freetype-2.3.9/builds/unix/freetype-config.in
===================================================================
--- freetype-2.3.9.orig/builds/unix/freetype-config.in
+++ freetype-2.3.9/builds/unix/freetype-config.in
@@ -101,11 +101,11 @@ if test "$local_prefix" = "yes" ; then
 fi
 
 if test "$echo_prefix" = "yes" ; then
-  echo $prefix
+  echo ${SYSROOT}$prefix
 fi
 
 if test "$echo_exec_prefix" = "yes" ; then
-  echo $exec_prefix
+  echo ${SYSROOT}$exec_prefix
 fi
 
 if test "$exec_prefix_set" = "yes" ; then
@@ -118,22 +118,22 @@ else
 fi
 
 if test "$echo_ft_version" = "yes" ; then
-  major=`grep define $includedir/freetype2/freetype/freetype.h \
+  major=`grep define ${SYSROOT}$includedir/freetype2/freetype/freetype.h \
          | grep FREETYPE_MAJOR \
          | sed 's/.*[ 	]\([0-9][0-9]*\).*/\1/'`
-  minor=`grep define $includedir/freetype2/freetype/freetype.h \
+  minor=`grep define ${SYSROOT}$includedir/freetype2/freetype/freetype.h \
          | grep FREETYPE_MINOR \
          | sed 's/.*[ 	]\([0-9][0-9]*\).*/\1/'`
-  patch=`grep define $includedir/freetype2/freetype/freetype.h \
+  patch=`grep define ${SYSROOT}$includedir/freetype2/freetype/freetype.h \
          | grep FREETYPE_PATCH \
          | sed 's/.*[ 	]\([0-9][0-9]*\).*/\1/'`
   echo $major.$minor.$patch
 fi
 
 if test "$echo_cflags" = "yes" ; then
-  cflags="-I$includedir/freetype2"
-  if test "$includedir" != "/usr/include" ; then
-    echo $cflags -I$includedir
+  cflags="-I${SYSROOT}$includedir/freetype2"
+  if test "${SYSROOT}$includedir" != "/usr/include" ; then
+    echo $cflags -I${SYSROOT}$includedir
   else
     echo $cflags
   fi
@@ -145,8 +145,8 @@ if test "$echo_libs" = "yes" ; then
     eval "rpath=\"$hardcode_libdir_flag_spec\""
   fi
   libs="-lfreetype @LIBZ@ @FT2_EXTRA_LIBS@"
-  if test "$libdir" != "/usr/lib" && test "$libdir" != "/usr/lib64"; then
-    echo -L$libdir $rpath $libs
+  if test "${SYSROOT}$libdir" != "/usr/lib" && test "${SYSROOT}$libdir" != "/usr/lib64"; then
+    echo -L${SYSROOT}$libdir $libs
   else
     echo $libs
   fi
@@ -154,7 +154,7 @@ fi
 
 if test "$echo_libtool" = "yes" ; then
   convlib="libfreetype.la"
-  echo $libdir/$convlib
+  echo ${SYSROOT}$libdir/$convlib
 fi
 
 # EOF
